gyp/pylib/gyp/xcodeproj_file.py

```diff
@@ -155,6 +155,9 @@
   _new_sha1 = sha.new
 
 
+if sys.version_info >= (3, 0):  # just easier
+    unicode = str
```

Comment:
On this one, can we please follow Python porting best practice [__use feature detection instead of version detection__](https://docs.python.org/3/howto/pyporting.html#use-feature-detection-instead-of-version-detection)?

Reply:
There's a reason for it in this case: exception -> substitution will run much slower than knowlege of version where this name doesn't exist. This is also used in `six` and works well.

Reply:
so I'd prefer to leave this kind of check for types and for a dictionary